src/script/api/script_news.hpp

```diff
@@ -24,25 +24,25 @@ class ScriptNews : public ScriptObject {
 	 */
 	enum NewsType {
 		/* Arbitrary selection of NewsTypes which might make sense for scripts */
-		NT_ACCIDENT          = ::NT_ACCIDENT,         ///< Category accidents.
-		NT_COMPANY_INFO      = ::NT_COMPANY_INFO,     ///< Category company info.
-		NT_ECONOMY           = ::NT_ECONOMY,          ///< Category economy.
-		NT_ADVICE            = ::NT_ADVICE,           ///< Category vehicle advice.
-		NT_ACCEPTANCE        = ::NT_ACCEPTANCE,       ///< Category acceptance changes.
-		NT_SUBSIDIES         = ::NT_SUBSIDIES,        ///< Category subsidies.
-		NT_GENERAL           = ::NT_GENERAL,          ///< Category general.
+		NT_ACCIDENT          = to_underlying(::NewsType::Accident),         ///< Category accidents.
+		NT_COMPANY_INFO      = to_underlying(::NewsType::CompanyInfo),      ///< Category company info.
+		NT_ECONOMY           = to_underlying(::NewsType::Economy),          ///< Category economy.
+		NT_ADVICE            = to_underlying(::NewsType::Advice),           ///< Category vehicle advice.
+		NT_ACCEPTANCE        = to_underlying(::NewsType::Acceptance),       ///< Category acceptance changes.
+		NT_SUBSIDIES         = to_underlying(::NewsType::Subsidies),        ///< Category subsidies.
+		NT_GENERAL           = to_underlying(::NewsType::General),          ///< Category general.
```

Comment:
You could also do `static constexpr NewsType NT_GENERAL = NewsType::General;` and get rid of the enumeration in the script API. That's what I have been doing lately.

Reply:
Hmm... good point. Didn't think about that. Must say the ones I changed were all invalid pool IDs, so that they become a singular constant isn't that big of a deal.